tests/cnf/core/network/metallb/tests/frrk8-tests.go

```diff
@@ -527,20 +532,30 @@ var _ = Describe("FRR", Ordered, Label(tsparams.LabelFRRTestCases), ContinueOnFa
 		It("Verify Frrk8 iBGP multihop over a secondary interface",
 			reportxml.ID("75248"), func() {
 
-				By("Collecting interface and VLAN information to create the secondary interface")
+				By("Collecting SR-IOV interface for secondary network")
 				srIovInterfacesUnderTest, err := NetConfig.GetSriovInterfaces(1)
 				Expect(err).ToNot(HaveOccurred(), "Failed to retrieve SR-IOV interfaces for testing")
 
-				vlanID, err := NetConfig.GetVLAN()
-				Expect(err).ToNot(HaveOccurred(), "Fail to set vlanID")
+				By("Assign secondary IP address to base interface on worker node 0")
+				createSecondaryIPOnBaseInterface("sec-int-worker0", workerNodeList[0].Definition.Name,
+					srIovInterfacesUnderTest[0], frrNodeSecIntIPv4Addresses[0], "2001:100::254")
 
-				By("create a secondary IP address on the worker node 0")
-				createSecondaryInterfaceOnNode("sec-int-worker0", workerNodeList[0].Definition.Name,
-					srIovInterfacesUnderTest[0], frrNodeSecIntIPv4Addresses[0], "2001:100::254", vlanID)
+				By("Assign secondary IP address to base interface on worker node 1")
+				createSecondaryIPOnBaseInterface("sec-int-worker1", workerNodeList[1].Definition.Name,
+					srIovInterfacesUnderTest[0], frrNodeSecIntIPv4Addresses[1], "2001:100::253")
 
-				By("create a secondary IP address on the worker node 1")
-				createSecondaryInterfaceOnNode("sec-int-worker1", workerNodeList[1].Definition.Name,
-					srIovInterfacesUnderTest[0], frrNodeSecIntIPv4Addresses[1], "2001:100::253", vlanID)
+				secondaryInterfaceName := srIovInterfacesUnderTest[0]
+
+				By("Verify secondary interfaces are UP with IP addresses on worker nodes")
+				for _, workerNode := range workerNodeList {
+					Eventually(func() error {
+						return checkInterfaceExistsOnNode(workerNode.Definition.Name, secondaryInterfaceName)
+					}, time.Minute, 5*time.Second).Should(Succeed(),
+						"Interface %s not ready on node %s", secondaryInterfaceName, workerNode.Definition.Name)
+				}
+
+				By("Waiting for interfaces to fully stabilize before creating macvlan NADs")
+				time.Sleep(10 * time.Second)
```

Comment:
Pleas find a method to check without timeout or remove it

Reply:
removed it - redundant